src/Connections/IConnection.ts

```diff
@@ -63,6 +63,8 @@ export interface IConnection {
     /**
      * If supported, this is sent when a user attempts to remove the connection from a room. The connection
      *  state should be removed and any resources should be cleaned away.
+     * @props purgeRemoteConfig Should the remote configuration for the connection be purged (in the case that
+     * other connections may be sharing a remote resource).
```

Comment:
What does this have to do with anything?
Is that a `param` that's not in the function's signature?